src/FlaUI.Core.UnitTests/Logging/AbstractBaseLoggerTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using FlaUI.Core.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Globalization;
+
+namespace FlaUI.Core.UnitTests.Logging
```

Comment:
Well the reason I do it is because if you are writing unit tests for a assembly it is nice to mirror the assembly structure withing the unit test assembly for organization, and if you go a step further and change the default namespace of your test assemblies this mirroring means that you don't need to an additional using. Which makes a lot of sense from a name-spacing perspective as the tests are really part of the code they test. People split them into their own assembly only so that they don't have to ship the tests with their production code.